src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -491,14 +492,17 @@ The .NET Foundation licenses this file to you under the MIT license.
         strip -no_code_signature_warning $(_StripFlag) &quot;$(NativeBinary)&quot;" />
   </Target>
 
-  <!-- NativeAOT-LLVM: separate target to reduce conflicts -->
-  <Target Name="LinkNativeLlvm"
-      Inputs="@(NativeObjects);@(NativeLibrary)"
-      Outputs="$(NativeBinary)"
-      DependsOnTargets="$(LinkNativeDependsOn)">
+  <Import Project="$(MSBuildThisFileDirectory)DotNetJsApi.targets" Condition="'$(_targetArchitecture)' == 'wasm' and '$(DotNetJsApi)' == 'true'" />
 
+  <Target Name="SetupLlvmLinker">
```

Comment:
Why was it required to split `LinkNativeLlvm`?

Reply:
Flags from `DotNetJsApi.targets` had to be applied after default ones. I don't remember exact details

Reply:
Here is a comparison if DotnetJs args are before/after default ones [dotnet.native.zip](https://github.com/dotnet/runtimelab/files/13671284/dotnet.native.zip)

Reply:
Ook, removing `pal_random.lib.js` from linking when `DotnetJsApi` solved it (we have it in `crypto.ts`)